h2o-core/src/test/java/water/util/VecUtilsTest.java

```diff
@@ -32,4 +34,25 @@ public void testStringVec2Categorical() {
       f.delete();
     }
   }
+
+  @Test
+  public void collectIntegerDomain() {
```

Comment:
I wrote this test to test collectIntegerDomain works. Why not keep it ? :question: